bin/compile

```diff
@@ -93,6 +93,11 @@ fi
 
 speak "Running stack"
 cd "$WORK_DIR"
+# Makefile support (if you need other build tools than Stack that you need to use in conjunction)
+if [ -f $BUILD_DIR/Makefile ]; then
+    speak "Makefile detected"
+    make -C $BUILD_DIR
+fi
```

Comment:
I was specifically concerned about cases like the https://github.com/haskell-servant/example-servant-elm case I mentioned above... here, the Makefile specifies the build command as just `stack build`, not `stack build --copy-bins`, and the subsequent steps in the buildpack fail.

For cases like these, we are treating the **Makefile as being at fault** for overriding the default action with something that will not let the buildpack, right? If so, I think I'll add a little note about how the build commands for Stack specified in the Makefile must include the `--copy-bins` flag in order to be compatible with the default expected behavior. Is that all right?